skypy/galaxies/vdf_sampler.py

```diff
@@ -0,0 +1,92 @@
+"""Sample from velocity distribution function
+"""
+
+import numpy as np
+import scipy.special as special
+from scipy.interpolate import interp1d
+
+
+def sample_vdf(x_min, x_max, resolution=100, size=1):
```

Comment:
Please do not use the variable `x` here. I suggest `vd_min`/`vd_max` or just `sigma_min` and `sigma_max`. I prefer `vd` as it makes it a bit clearer. 

Also similar to the examples the default of size should be `None` and in which case the function should return one value.

Reply:
Done.